src/Notification/Warning.js

```diff
@@ -0,0 +1,117 @@
+/* @flow */
+
+import * as React from 'react';
+import { Animated, View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { Icon } from '../Icon';
+import Touchable from '../Button/Touchable';
+import StyleSheet from '../PlatformStyleSheet';
+
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+import type { NotificationStyleType } from '../types';
+
+import Text from '../Text';
+
+type Props = {|
+  +style?: StylePropType,
+  +notificationStyle: NotificationStyleType,
+  +warningTitle: React.Node | string,
+  +warningMessage: React.Node | string,
+  +onPress: () => void,
+|};
+
+const AnimatedTouchable = Animated.createAnimatedComponent(Touchable);
+
+export default function Warning({
```

Comment:
Question, why this component is `Warning` but it can be `error` or `success` ? Isn't that a bit confusing ? Or web one in `orbit-components` is the same?

Reply:
Currently you have to specify what type of `Notification` (`alert` or `warning`) you would like to use ( it's based on `figma` design). Then you have to specify notification style (`error`, `warning` or `success`.

Reply:
I could not find any similar component on `orbit-components`. 
I can see why the name could be confusing. 
Maybe we could have ImportantNotification and InformativeNotification. I don't know. What do you think?